shared/style/action_menu.css

```diff
@@ -238,7 +238,7 @@
   font-style: italic;
 }
 
-[role="dialog"][data-type="action"] > menu > button:last-child {
+[role="dialog"][data-type="action"] > menu > button.last {
```

Comment:
mmm I'm afraid that there are forms in gaia that use this selector, but don't have ```last``` selector or you have a plan to update every place?